geoviews/element/geo.py

```diff
@@ -214,6 +215,28 @@ class Image(_Element, HvImage):
     group = param.String(default='Image')
 
 
+class RGB(_Element, HvRGB):
```

Comment:
Couldn't ``_Element`` have been called ``CartElement`` or similar? It has ``_abstract=True`` so I don't see the need for the underscore...